QGS GUI: minor language corrections/improvements (#3387)
qgis/QGIS-Documentation: source/docs/user_manual/introduction/qgis_gui.rst

3D visualization support is offered through the 3D map view.

.. note::
   3D visualization in QGIS requires a recent version of the

>    QT library (5.8 or later).

Author: WHat's the official minimum Qt version for 3.4? If it's >=5.8, can we remove this note?

Reviewer: Based on http://osgeo-org.1560.x6.nabble.com/QGIS-Developer-Min-qt-version-for-3-6-tt5382706.html I think it's lower than 5.8. The idea of the note is to let users know that it's not because they have 3.4 than they necessarily will have operating 3D. Depends on underlying library/OS. (That said is 5.8 still **recent**?)

Author: OK, we'd best leave it in. Thanks.

associated with the item you are looking for (name, tag, keyword...) and you get
a list that updates as you write. You can also limit the search scope using
:ref:`locator filters <locator_options>`. Click the |search| button to
select any of them and press the :guilabel:`Configure` entry for global settings.

In the area next to the locator bar, a summary of actions you've carried out will
be shown when needed  (such as selecting features in a layer, removing layer) or
a long description of the tool you are hovering over (not available for all tools).

In case of lengthy operations, such as gathering of statistics in raster layers,
executing Processing algorithms or rendering several layers in the map view, a
progress bar is displayed in the status bar.

The |tracking| :guilabel:`Coordinate` option shows the current position of the mouse,
following it while moving across the map view. You can set the units (and precision)
in the :menuselection:`Project --> Properties... --> General` tab.
Click on the small button at the left of the textbox to toggle between
the Coordinate option and the |extents| :guilabel:`Extents` option that displays

> the coordinates of the current lower leftmost and upper rightmost

Author: Actually, it should be:

```suggestion
the coordinates of the current bottom-left and top-right
```

> the coordinates of the current lower leftmost and upper rightmost

Reviewer 2: Just do it! :-)

> corner of the map view in map units.

Reviewer 2: ```suggestion
corners of the map view in map units.
```

Reviewer 2: Perhaps you forgot to change to plural?

a list that updates as you write. You can also limit the search scope using
:ref:`locator filters <locator_options>`. Click the |search| button to
select any of them and press the :guilabel:`Configure` entry for global settings.

In the area next to the locator bar, a summary of actions you've carried out will
be shown when needed  (such as selecting features in a layer, removing layer) or
a long description of the tool you are hovering over (not available for all tools).

In case of lengthy operations, such as gathering of statistics in raster layers,
executing Processing algorithms or rendering several layers in the map view, a
progress bar is displayed in the status bar.

The |tracking| :guilabel:`Coordinate` option shows the current position of the mouse,
following it while moving across the map view. You can set the units (and precision)
in the :menuselection:`Project --> Properties... --> General` tab.
Click on the small button at the left of the textbox to toggle between
the Coordinate option and the |extents| :guilabel:`Extents` option that displays
the coordinates of the current lower leftmost and upper rightmost

> points of the map view in map units.

Reviewer 2: ```suggestion
corner of the map view in map units.
```

Reviewer 2: Or pixel?

Next to the coordinate display you will find the :guilabel:`Scale` display.
It shows the scale of the map view. There is a scale selector, which allows you to

> choose between :ref:`predefined and custom scales <predefinedscales>` to apply to

Reviewer 2: ```suggestion
choose between :ref:`predefined and custom scales <predefinedscales>`.
```

> the map view.

Reviewer 2: ```suggestion
```
